modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeStatus.java

```diff
@@ -62,34 +58,30 @@ public VisorRollingUpgradeStatus() {
      * @param forcedModeEnabled {@code true} if forced mode is enabled.
      * @param initVer Initial version.
      * @param targetVer Resulting version.
-     * @param supportedFeatures Feature set that is supported by nodes.
      */
     public VisorRollingUpgradeStatus(
         boolean enabled,
         boolean forcedModeEnabled,
         String initVer,
-        String targetVer,
-        Set<String> supportedFeatures
+        String targetVer
     ) {
         this.enabled = enabled;
         this.forcedModeEnabled = forcedModeEnabled;
         this.initVer = initVer;
         this.targetVer = targetVer;
-        this.supportedFeatures = supportedFeatures;
     }
 
     /**
      * Creates a wrapper class that represents a value of {@link RollingUpgradeStatus}.
      *
      * @param status Rolling upgrade status to be copied.
      */
-    public VisorRollingUpgradeStatus(RollingUpgradeStatus status) {
+    public VisorRollingUpgradeStatus(IgniteRollingUpgradeStatus status) {
```

Comment:
It seems that we can use RollingUpgradeStatus here. This will allow avoiding the following:
`status = new VisorRollingUpgradeStatus((IgniteRollingUpgradeStatus)changeModeRes.status());`
(see `VisorRollingUpgradeChangeModeResult`)
What do you think?